kotlin/gradle/libs.versions.toml

```diff
@@ -35,7 +35,7 @@ mockito-core = { group = "org.mockito", name="mockito-core", version.ref= "mocki
 mockito-inline = { group = "org.mockito", name="mockito-inline", version.ref= "mockito"}
 mockito-kotlin = { group = "org.mockito.kotlin", name="mockito-kotlin", version.ref= "mockitoKotlin"}
 squareup-okhttp= { group = "com.squareup.okhttp3", name="okhttp", version.ref= "okhttp"}
-pixelpass= { group = "io.mosip", name="pixelpass", version.ref= "pixelpass"}
+pixelpass= { group = "io.mosip", name="pixelpass-aar", version.ref= "pixelpass"}
```

Comment:
shouldn't we use JAR as this library will be used by mimoto backend.

Reply:
Since it is transitive dependency, it should bundle the whole injivcrenderer as jar right. Moreover pixelpass-jar is already used in mimoto, will it conflicts the dependency if pixelpass-jar is used here ?

Reply:
pixelpass-jar will be removed from mimoto once we are moved to svg template as template will be generating the QR code.
When this inji renderer jar will be used by mimoto, during qr code generation it might fail as aar won't get executed by mimoto

Reply:
done , updated to pixelpass-jar